### DISCUSSION TITLE
Feature Request: Order title & buttons and Windows inside windows

### DISCUSSION THREAD

**Discussion by TaromaruYuki** [nextapps-de/winbox#129] [Ideas]
This is an amazing library, and it so much better than what I have made for windows. I'm making a Windows 3.1-like look (See the first image) and I need to reorder the title and the buttons. Maybe when you make the box, you can specify what order you want them in. (See example code)

![image](https://user-images.githubusercontent.com/48397556/121762698-b365cb80-cb05-11eb-8c19-4487e170a61a.png)

Example Code:
```js
new WinBox("Program Manager", {
    header_order: ["close", "title", "max", "min"]
});
```

Also having windows inside windows would be a good feature too, as this is done with the program manager inside Windows 3.1

![image](https://user-images.githubusercontent.com/48397556/121762790-2111f780-cb06-11eb-8934-25f687e0abae.png)

**Comment by ts-thomas** [comment]
Custom templates are now supported.
